### ISSUE TITLE
Python 2.7.6.

### ISSUE THREAD

**Issue by textbrowser** [maandree/unstickpixels#2]
If possible, please correct. If not, please abort the program. Successfully tested with Python 3.4.3.

```
  File "./unstickpixels", line 3
SyntaxError: Non-ASCII character '\xe2' in file ./unstickpixels on line 3, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
```

**Comment by maandree** [comment]
I will fix this at the same this as issue #1. I have not decided whether it should still be written in Python 3 or if I should rewrite it to C since there is not tangible benefit to using Python for this program.

**Comment by textbrowser** [comment]
Alright.
